clean_project/search_api.py

```diff
@@ -0,0 +1,45 @@
+"""
+Search a word in a file and return all the occurrences specified the line numbers
+"""
+import argparse
+from collections import defaultdict
+from flask import Flask, jsonify
+import search_engine
+
+BOLD = '\033[1m'
+END = '\033[0m'
+
+
+app = Flask(__name__)
+
+
+def config_app(search_object):
+    app.config['search'] = search_object
+    return app
+
+
+@app.route('/search/<string:search_word>', methods=['GET'])
+def get_request(search_word):
+    """
+    Get request to verify if a word exist in a database
```

Comment:
that's not what this function does no? it doesn't verify it exists, it returns the results of a search operation